### PR TITLE
now stores messages it receives and stacks them in the notification bar

### PR DESCRIPTION
**Pull request by edewit**
How to test:
* create a cordova project with push
* add the plugin via git:
```cmd
$> corodva plugin add  https://github.com/edewit/aerogear-pushplugin-cordova#stack-messages
```
Put the app in background send couple of messages and see the them stacked in the notification bar

### CODE REVIEW

**Comment by edewit** [comment]
this fixes [AGCORDOVA-113](https://issues.jboss.org/browse/AGCORDOVA-113) introduces a dependency on aerogear-android-store @danielpassos could you have a look there might be a better way to do this.

File: src/android/org/jboss/aerogear/cordova/push/Message.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import android.os.Bundle;
+
+import org.jboss.aerogear.android.core.RecordId;
+
+import java.util.UUID;
+
+/**
+ * Message
+ */
+public class Message {
```
Comment (matzew) [inline src/android/org/jboss/aerogear/cordova/push/Message.java]: Perhaps `Stacked/StoredMessage` ?

File: src/android/org/jboss/aerogear/cordova/push/Message.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
+package org.jboss.aerogear.cordova.push;
```
Comment (matzew) [inline src/android/org/jboss/aerogear/cordova/push/Message.java]: license header

**Comment by matzew** [comment]
nice idea, @edewit :tada:

**Comment by keithdmoore** [comment]
Love it!!!!

**Comment by danielpassos** [comment]
:eyes:

**Comment by danielpassos** [comment]
:+1: 

```shell
[passos]: ~/Desktop
→ git clone https://github.com/jboss-mobile/unified-push-helloworld.git
Cloning into 'unified-push-helloworld'...
...
```

```shell
[passos]: ~/Desktop
→ cd unified-push-helloworld/cordova/
→ open www/push-config.json
```

```shell
[passos]: ~/Desktop/unified-push-helloworld/cordova [git:master]
→ cordova platform add android
Adding android project...
Running command: /Users/passos/.cordova/lib/npm_cache/cordova-android/4.1.1/package/bin/create /Users/passos/Desktop/unified-push-helloworld/cordova/platforms/android org.jboss.aerogear.unifiedpush.HelloWorld "AeroGear UnifiedPush HelloWorld" --cli
Creating Cordova project for the Android platform:
	Path: platforms/android
	Package: org.jboss.aerogear.unifiedpush.HelloWorld
	Name: AeroGear UnifiedPush HelloWorld
	Activity: MainActivity
	Android target: android-22
Copying template files...
Android project created with cordova-android@4.1.1
Running command: /Users/passos/Desktop/unified-push-helloworld/cordova/hooks/after_prepare/icon.js /Users/passos/Desktop/unified-push-helloworld/cordova
```

```shell
[passos]: ~/Desktop/unified-push-helloworld/cordova [git:master] *
→ cordova plugin add https://github.com/edewit/aerogear-pushplugin-cordova#stack-messages
Fetching plugin "https://github.com/edewit/aerogear-pushplugin-cordova" via git clone
Repository "https://github.com/edewit/aerogear-pushplugin-cordova" checked out to git ref "stack-messages".
Installing "aerogear-cordova-push" for android
Fetching plugin "https://github.com/vstirbu/PromisesPlugin.git" via git clone
Repository "https://github.com/vstirbu/PromisesPlugin.git" checked out to git ref "master".
Installing "es6-promise-plugin" for android
```

```shell
[passos]: ~/Desktop/unified-push-helloworld/cordova [git:master] *
→ cordova run android
Running command: /Users/passos/Desktop/unified-push-helloworld/cordova/hooks/after_prepare/icon.js /Users/passos/Desktop/unified-push-helloworld/cordova
Running command: /Users/passos/Desktop/unified-push-helloworld/cordova/platforms/android/cordova/run
ANDROID_HOME=/Users/passos/Development/SDK/android-sdk-macosx
JAVA_HOME=/Library/Java/JavaVirtualMachines/jdk1.8.0_20.jdk/Contents/Home/
WARNING : No target specified, deploying to device '192.168.60.101:5555'.
Running: /Users/passos/Desktop/unified-push-helloworld/cordova/platforms/android/gradlew cdvBuildDebug -b /Users/passos/Desktop/unified-push-helloworld/cordova/platforms/android/build.gradle -PcdvBuildArch=x86 -Dorg.gradle.daemon=true

.
.
.
BUILD SUCCESSFUL

Total time: 4 mins 16.196 secs
Built the following apk(s):
    /Users/passos/Desktop/unified-push-helloworld/cordova/platforms/android/build/outputs/apk/android-debug.apk
Using apk: /Users/passos/Desktop/unified-push-helloworld/cordova/platforms/android/build/outputs/apk/android-debug.apk
Installing app on device...
Launching application...
LAUNCH SUCCESS
```
screenshot 2015-11-13 10 43 38

**Pull request merged**